libsolidity/lsp/FileRepository.h

```diff
@@ -0,0 +1,53 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+#pragma once
+
+#include <libsolidity/interface/FileReader.h>
+
+#include <string>
+#include <map>
+
+namespace solidity::lsp
+{
+
+class FileRepository
+{
+public:
+	explicit FileRepository(boost::filesystem::path const& _basePath = "/"):
```

Comment:
I thought we agreed to drop the default for that? I see `/` as a temporary hack that we'll replace with something better after we get the MVP out so I'd really prefer if it had to be used explicitly, preferably with a comment explaining why we're doing it.
```suggestion
	explicit FileRepository(boost::filesystem::path const& _basePath):
```